.github/workflows/ci.yml

```diff
@@ -19,6 +22,8 @@ jobs:
           - task: build-storybooks
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
What does this do?

Reply:
By default the checkout action only fetches a single commit. `fetch-depth: 0` gets the full Git history. The Chromatic action needs this to keep track of changes so I assume it's the same for nx caching.

Reply:
yeah thats my understanding